Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -648,107 +651,123 @@ define([
+                return this.properties.dayTextureContrast;
             },
             u_dayTextureHue : function() {
-                return this.dayTextureHue;
+                return this.properties.dayTextureHue;
             },
             u_dayTextureSaturation : function() {
-                return this.dayTextureSaturation;
+                return this.properties.dayTextureSaturation;
             },
             u_dayTextureOneOverGamma : function() {
-                return this.dayTextureOneOverGamma;
+                return this.properties.dayTextureOneOverGamma;
             },
             u_dayIntensity : function() {
-                return this.dayIntensity;
+                return this.properties.dayIntensity;
             },
             u_southAndNorthLatitude : function() {
-                return this.southAndNorthLatitude;
+                return this.properties.southAndNorthLatitude;
             },
             u_southMercatorYAndOneOverHeight : function() {
-                return this.southMercatorYAndOneOverHeight;
+                return this.properties.southMercatorYAndOneOverHeight;
             },
             u_waterMask : function() {
-                return this.waterMask;
+                return this.properties.waterMask;
             },
             u_waterMaskTranslationAndScale : function() {
-                return this.waterMaskTranslationAndScale;
+                return this.properties.waterMaskTranslationAndScale;
             },
             u_minMaxHeight : function() {
-                return this.minMaxHeight;
+                return this.properties.minMaxHeight;
             },
             u_scaleAndBias : function() {
-                return this.scaleAndBias;
+                return this.properties.scaleAndBias;
             },
 
 
-            initialColor : new Cartesian4(0.0, 0.0, 0.5, 1.0),
-            zoomedOutOceanSpecularIntensity : 0.5,
-            oceanNormalMap : undefined,
-            lightingFadeDistance : new Cartesian2(6500000.0, 9000000.0),
-
-            center3D : undefined,
-            modifiedModelView : new Matrix4(),
-            tileRectangle : new Cartesian4(),
-
-            dayTextures : [],
-            dayTextureTranslationAndScale : [],
-            dayTextureTexCoordsRectangle : [],
-            dayTextureAlpha : [],
-            dayTextureBrightness : [],
-            dayTextureContrast : [],
-            dayTextureHue : [],
-            dayTextureSaturation : [],
-            dayTextureOneOverGamma : [],
-            dayIntensity : 0.0,
-
-            southAndNorthLatitude : new Cartesian2(),
-            southMercatorYAndOneOverHeight : new Cartesian2(),
-
-            waterMask : undefined,
-            waterMaskTranslationAndScale : new Cartesian4(),
-
-            minMaxHeight : new Cartesian2(),
-            scaleAndBias : new Matrix4()
+            // make a separate object so that changes to the properties are seen on
```

Comment:
This is because we want a shallow copy here, right?  Let's make the comment explicit.

In general, we might want to think about this for derived commands since this code needs to know an awful lot about how they work in order to get this right, maybe this is part of a bigger uniform refactor for uniform buffers, auto packing, etc.